conf/bootstrap_apt

```diff
@@ -17,6 +17,30 @@ fatal() {
     exit 1
 }
 
+build_dummy_pkg() {
+    PKG=$1
+    mkdir -p /tmp/$PKG/DEBIAN
+    PKG_V="2:${PHP_VERSION}"
```

Comment:
PKG and PKG_V could be locals here `local PKG=$1 PKG_VER=...` or `declare PKG=$1 PKG_VER=...`

Reply:
Great point! Thanks man. I'll apply that change then merge this so we don't hold up Mattie.